specs/Separate_conf_files.adoc

```diff
@@ -0,0 +1,141 @@
+the information into single object.
+
+== Use Cases
+
+None
+
+== Proposed change
+
+** Modify the sample configuration files in each modules. It should contains its own
+   and relevant configuration information only.
+** Rename the sample configuration files in each modules with proper name.
+** In common repository modify spec file to create a new directory inside /etc/tendrl/ in
+   the name of conf.d.
+** Modify the spec file in each modules to copy the sample configuration file into proper
+   directory:
+      * From common repository modify the spec file to copy the sample configuration file
+        inside the directory /etc/tendrl.
+      * From gluster_integration, ceph_integration and node_agent modify the each spec files
+        to copy the sample configuration files inside the directory /etc/tendrl/conf.d.
+** Modify config.py from common repository to read each configuration files independently
+   and it should combine all the information into single object.
+** From node_agent repository modify copy statement in /doc/source/installation.rst
+   based on new configuration file path change.
+** From gluster_integration repository modify copy statement in /doc/source/installation.rst
+   based on new configuration file path change.
+** From ceph_integration repository modify copy statement /doc/source/installation.rst based
+   on new configuration file path change.
+
+== Alternatives
+
+None
+
+== Data model impact
+
+None
+
+== Impacted modules
+
+=== Tendrl/common impact
+
+** /doc/source/installation.rst
+** /etc/tendrl/tendrl.conf.sample
+** config.py
+** tendrl-common.spec
+
+=== Tendrl/node_agent impact
+
+** /doc/source/installation.rst
+** etc/tendrl/tendrl.conf.sample
+** config.py
+** tendrl-node-agent.spec
+
+=== Tendrl/gluster_integration impact
+
+** /doc/source/installation.rst
+** etc/tendrl/tendrl.conf.sample
+** config.py
+** tendrl-gluster-integration.spec
+
+=== Tendrl/ceph_integration impact
+
+** /doc/source/installation.rst
+** etc/tendrl/tendrl.conf.sample
+** config.py
+** tendrl-ceph-integration.spec
+
+
+== Security impact
+
+None
+
+== Notifications/Monitoring impact
+
+None
+
+== Other end user impact
+
+None
+
+== Performance Impact
+
+None
+
+== Other deployer impact
+
+None
+
+== Developer impact
+
+None
+
+== Implementation
+
+** github issue link here
+
+== Assignee(s)
+
+Primary assignee: TODO
+
+Other contributor(s): TODO
+
+== Work Items
+
+* Change sample configuration file in  gluster_integration, ceph_integration,
+  node_agent, common.
+* Change config.py to read all configuration file in common.
+* Change installation document in gluster_integration, ceph_integration,
+  node_agent, common.
+* Change spec file in gluster_integration, ceph_integration,
+  node_agent, common.
+
+== Dependencies
+
+* It is dependent on refactor job.
+
+== Testing
+
+* Check all configuration files are read successfully
```

Comment:
Also a sanity of one major flow should be performed as part of testing this I feel. Also make sure all the logging etc is happening properly.